centml/compiler/hidet_compiled_model.py

```diff
@@ -0,0 +1,76 @@
+import tempfile
+from typing import List, Sequence
+from torch import Tensor as TorchTensor
+from hidet import Tensor as HidetTensor
+from hidet.ir.type import DataType
+from hidet.ir.expr import SymbolVar
+from hidet.runtime import CompiledGraph, load_compiled_graph
+from hidet.cuda.graph import CudaGraphCreationError
+from hidet.ffi import runtime_api
+from hidet.graph.frontend.torch.dynamo_config import dynamo_config
+from hidet.graph.frontend.torch.dynamo_backends import preprocess_inputs, deserialize_output
+
+
+# This is a serializable version of hidet-org's get_wrapper function
+# pickle cannot serialize local functions, so get_wrapper's is rewritten as a class
+class HidetCompiledModel:
```

Comment:
I believe we can remove this now since your hidet changes have been merged to the public repo